### DISCUSSION TITLE
Is there a way to make the auto-post?

### DISCUSSION THREAD

**Discussion by mobilemintcarcare** [timlrx/tailwind-nextjs-starter-blog#605] [Q&A]
When the post date matches the current date and then publishes automatically?

**Comment by dlarroder** [comment]
I don't think there is any auto-post feature for this template at the moment, but feel free to create a script for that if you like

**Comment by dariusz-wozniak** [comment]
@mobilemintcarcare: You can filter posts by date.

For example, you can modify page.tsx:

```tsx
export default async function Page() {
  let sortedPosts = sortPosts(allBlogs)
  if (isProduction) {
    const today = new Date()
    today.setHours(12, 0, 0, 0)
    sortedPosts = sortedPosts.filter((post) => {
      return new Date(post.date) <= today
    })
  }
  const posts = allCoreContent(sortedPosts)
  return <Main posts={posts} />
}
```

Check that commit: https://github.com/dariusz-wozniak/dariuszwozniak.net-v2/commit/793eb421217a2bdc3d5f7dfe3118c1da52163064

**Reply by ray-android** [reply]
I'm just checking to see if there have been any recent updates on this matter. 

The suggestion from @dariusz-wozniak is good, but I believe we should also take into account the generation of pages, blogs, tags, and sitemap. However, concerning tags, it appears that a redeployment is necessary to ensure the latest updates are reflected. I'm uncertain about the most efficient way to handle this automatically. 

Any suggestions?
